packages/manager/modules/pci/src/projects/project/kubernetes/components/network-form/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,24 @@
+{
+  "kubernetes_network_form_gateway_label": "Stratégie de routage sortant",
+  "kubernetes_network_form_gateway_description": "Vous pouvez spécifier une passerelle (gateway) spécifique pour router le trafic de vos noeuds de travail (par exemple récupération d'images Docker depuis un registry).",
+  "kubernetes_network_form_gateway_public": "Interface publique des noeuds",
+  "kubernetes_network_form_gateway_public_description": "Chaque noeud utilisera sa propre interface publique",
+  "kubernetes_network_form_gateway_private": "Interface privée des noeuds",
+  "kubernetes_network_form_gateway_private_description_p1": "À selectionner si vous souhaitez utiliser une passerelle",
+  "kubernetes_network_form_gateway_private_description_p2": "ou Custom tel que configuré dans votre réseau privé",
+  "kubernetes_network_form_gateway_private_description_link": "OVHcloud Manager",
+  "kubernetes_network_form_gateway_vrack_field_ip_placeholder": "Saisir une adresse IP",
+  "kubernetes_network_form_gateway_vrack_field_ip_error_pattern": "Veuillez saisir une adresse IP valide",
+  "kubernetes_network_form_label": "Réseau privé attaché",
+  "kubernetes_network_form_none": "Aucun (IPs publiques)",
+  "kubernetes_network_form_description": "Par défaut, vos nœuds de travail disposent d'une IPv4 publique. Si vous choisissez un réseau privé, les IP publiques de ces nœuds seront utilisées exclusivement pour leur administration/le lien avec le control plane Kubernetes, et vos noeuds se verront attribuer une IP sur le vLAN du réseau privé choisi.",
```

Comment:
Is it "control plane" or "control panel" ?

Reply:
It's control plane 👍